src/Components/App/App.js

```diff
@@ -12,41 +11,42 @@ class App extends Component {
     super()
     this.state = {
       movies: [],
-      clicked: false
+      clicked: false,
+      singleMovieDetails: [],
+      err: ''
     }
   }
 
   componentDidMount() {
     fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
       .then(response => response.json())
       .then(data => this.setState({ movies: data.movies, clicked: false }))
-      // .then(err => this.setState({ err: err.message }))
-  }
-
-  componentWillUnmount() {
-
+      .catch(() => {this.setState({err: 'Refresh Page'})})
   }
 
   viewMovie = (id) => {
     console.log('CHECKING', this.state.movies)
     const findMovie = this.state.movies.find(movie => movie.id === id)
-    this.setState({ movies: [findMovie], clicked: true})
+    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${findMovie.id}`)
+    .then(response => response.json())
+    .then(data => this.setState({clicked: true, singleMovieDetails: data.movie}))
```

Comment:
Eventually you'll want to pull your fetch functionality into an apiCalls.js file and add error handling for not-ok responses to those fetch calls.  You can still invoke them in the same places in app.js, you'll just want to declare that functionality in apiCalls.js